### ISSUE TITLE
Feedback / Update Design 

### ISSUE THREAD

**Issue by Majed610** [Aliado-Leipzig/VNG#5]
Hi guys, 

here the feedback from the client: 
[BALANCE Screendesign Korrekturen V1-2.pdf](https://github.com/Aliado-Leipzig/VNG/files/8098620/BALANCE.Screendesign.Korrekturen.V1-2.pdf)

the updated XD: 
https://xd.adobe.com/view/8b4a0463-2872-440a-80f9-14b254ce0fdf-de19/

Please check the feedback and let me know if you have any question. By changes which require a lot of time, kindly advise me about the estimates so that i can check with the client. 

Best regards
Majed

**Comment by Mirsho** [comment]
Hi Majed,

Thank you so much for the feedback translation and clarifications.

The time estimations are:

- 6 hours for overall changes. (4 hours for replacing/positioning images and sections, 2 hours for general style rules changes).
- 2 hours for the dotted circle rolling animation on scroll.

I have a quick question about the animation, to check if I understand it properly:

![Roll on Scroll](https://user-images.githubusercontent.com/50135963/154975574-392f323b-7556-47d2-aebe-0e91dd1de088.png)

Basically when user scroll up and down, the circle spins with the mouse scroll like a gear, holding its position. Is that right?

Best regards,
Adrián.
